packages/smooth_app/lib/pages/onboarding/country_selector.dart

```diff
@@ -36,8 +37,20 @@ class _CountrySelectorState extends State<CountrySelector> {
 
   Future<void> _init() async {
     final String locale = Localizations.localeOf(context).languageCode;
-    final List<Country> localizedCountries =
-        await IsoCountries.iso_countries_for_locale(locale);
+    List<Country> localizedCountries;
+
+    try {
+      localizedCountries = await IsoCountries.iso_countries_for_locale(locale);
+    } on MissingPluginException catch (_) {
+      // Locales are not implemented on desktop and web
```

Comment:
```suggestion
      // Locales are not implemented on desktop and web
      // TODO: Add a complete list
```

Reply:
> Not sure this is relevant, as we won't release to desktop and web

Isn't that a PR about desktops?

Reply:
Yes exactly, but desktop & web are only to ease the development

Reply:
Btw web cannot work, it doesn't support sqflite.
Understood about desktop/dev, but we may want one day to make it available on desktops too (I even think there's at least one issue about that). So a `TODO` would be good enough for the moment.